core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -531,8 +531,12 @@ public AggregationData apply(MutableMean arg) {
       implements Function<MutableDistribution, AggregationData> {
     @Override
     public AggregationData apply(MutableDistribution arg) {
+      List<Long> boxedBucketCounts = new ArrayList<Long>();
+      for (long bucketCount : arg.getBucketCounts()) {
+        boxedBucketCounts.add(bucketCount);
+      }
       return DistributionData.create(arg.getMean(), arg.getCount(), arg.getMin(), arg.getMax(),
-          arg.getSumOfSquaredDeviations(), arg.getBucketCounts());
+          arg.getSumOfSquaredDeviations(), boxedBucketCounts);
```

Comment:
Doesn't MutableDistribution.getBucketCounts() return a List<Long>? Seems like we're going to do a lot of unnecessary copying (and boxing/unboxing).

Reply:
MutableDistribution.getBucketCounts() returns a long[], but we could change it since it isn't part of the API.  I thought it was more important to avoid the boxing/unboxing when stats are recorded and aggregated in MutableDistribution than when we return the aggregated data in a DistributionData, since stats should be queried much less frequently.